fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/common/GlobalConfigurationHelper.java

```diff
@@ -41,18 +41,17 @@ public GlobalConfigurationHelper(final RequestSpecification requestSpec, final R
 
     public static ArrayList<HashMap> getAllGlobalConfigurations(final RequestSpecification requestSpec,
             final ResponseSpecification responseSpec) {
-        final String GET_ALL_GLOBAL_CONFIG_URL = "/fineract-provider/api/v1/configurations?" + Utils.TENANT_IDENTIFIER;
+        final String getAllGlobalConfigURL = "/fineract-provider/api/v1/configurations?" + Utils.TENANT_IDENTIFIER;
         LOG.info("------------------------ RETRIEVING ALL GLOBAL CONFIGURATIONS -------------------------");
-        final HashMap<String, ArrayList<HashMap>> response = Utils.performServerGet(requestSpec, responseSpec, GET_ALL_GLOBAL_CONFIG_URL,
-                "");
+        final HashMap<String, ArrayList<HashMap>> response = Utils.performServerGet(requestSpec, responseSpec, getAllGlobalConfigURL, "");
         return response.get("globalConfiguration");
     }
 
     public static HashMap getGlobalConfigurationById(final RequestSpecification requestSpec, final ResponseSpecification responseSpec,
             final String configId) {
-        final String GET_GLOBAL_CONFIG_BY_ID_URL = "/fineract-provider/api/v1/configurations/" + configId + "?" + Utils.TENANT_IDENTIFIER;
+        final String getGlobalConfigByIDurl = "/fineract-provider/api/v1/configurations/" + configId + "?" + Utils.TENANT_IDENTIFIER;
         LOG.info("------------------------ RETRIEVING GLOBAL CONFIGURATION BY ID -------------------------");
```

Comment:
Would be good to be consistent at capitalising URL - so this would be getGlobalConfigByIdURL

Reply:
updated